api/src/pcapi/core/finance/api.py

```diff
@@ -1378,12 +1381,22 @@ def get_individual_data(query: BaseQuery) -> BaseQuery:
                 offerers_models.Offerer.name.label("offerer_name"),
                 offerers_models.Offerer.siren.label("offerer_siren"),
                 models.Deposit.type.label("deposit_type"),
-                sqla_func.sum(models.Pricing.amount).label("pricing_amount"),
+                sa.case((offerers_models.Offerer.is_caledonian == True, "NC"), else_=None).label("is_caledonian"),
```

Comment:
Dès lors que c'est une colonne « is caledonian », pourquoi pas True / False à la place de "NC" / None ?

Reply:
C'est plus pratique pour remplir la colonne excel ensuite

Reply:
Alors la colonne est mal nommée :)